include/deal.II/fe/fe_base.h

```diff
@@ -391,6 +398,13 @@ class FiniteElementData
   unsigned int
   n_dofs_per_cell() const;
 
+  /**
+   * Number of global dofs per cell, degrees of freedom which are not within
+   * this cell. (Catmull-Clark subd)
```

Comment:
~~(Catmull-Clark subd)~~

Reply:
This should also be documented better (copy stuff from above?)